dot/rpc/modules/api_mocks.go

```diff
@@ -46,7 +46,7 @@ func NewMockBlockAPI() *modulesmocks.BlockAPI {
 // NewMockCoreAPI creates and return an rpc CoreAPI interface mock
 func NewMockCoreAPI() *modulesmocks.CoreAPI {
 	m := new(modulesmocks.CoreAPI)
-	m.On("InsertKey", mock.AnythingOfType("crypto.Keypair"))
+	m.On("InsertKey", mock.AnythingOfType("crypto.Keypair"), mock.AnythingOfType("string")).Return(nil)
```

Comment:
nit & out of scope: we should avoid using `mock.AnythingOfType` I've seen it multiple times cause too-flexible tests letting bugs slip through. It looks like we use it pretty much everywhere so that's that, but we should not for future new mocks/tests.